runtime/profile/_/mod/core/data/dialog/unique/lily.lua

```diff
@@ -0,0 +1,164 @@
+               GUI.play_sound("core.complete1")
+               GUI.txt(I18N.get("core.locale.common.something_is_put_on_the_ground"))
+               GUI.show_journal_update_message()
+            end,
+            {"progress.last._2"},
+         },
+         choices = {
+            {"__END__", "__MORE__"}
+         },
+         on_finish = function(t)
+            Internal.set_quest_flag("pael_and_her_mom", 1000)
+            t.speaker:set_flag("IsSilent", true)
+         end
+      },
+
+      festival = {
+         text = {
+            {"progress.festival.dialog"},
+         },
+         choices = function()
+            local choices = {}
+            local potion = Item.find("core.potion_of_cure_corruption", "PlayerInventory")
+            if potion ~= nil then
+               table.insert(choices, {"festival_give", "progress.festival.choices.give"})
+            end
+            table.insert(choices, {"festival_take_care", "progress.festival.choices.take_care"})
+
+            return choices
+         end
+      },
+      festival_give = {
+         text = {
+            give_potion,
+            {"progress.festival.give.dialog"},
+         }
+      },
+      festival_take_care = {
+         text = {
+            {"progress.festival.take_care"},
+         }
+      },
+
+      end_life = {
+         text = {
+            {"progress.end_life.dialog._0", args = function() return {Chara.player().basename} end},
+            {"progress.end_life.dialog._1"},
+         },
+         choices = function()
+            local choices = {}
+            local potion = Item.find("core.potion_of_cure_corruption", "PlayerInventory")
+            if potion ~= nil then
+               table.insert(choices, {"end_life_give", "progress.end_life.choices.give"})
+            end
+            table.insert(choices, {"end_life_end", "progress.end_life.choices.end"})
+            table.insert(choices, {"end_life_leave", "progress.end_life.choices.leave"})
+
+            return choices
+         end
+      },
+      end_life_give = {
+         text = {
+            give_potion,
+            {"progress.end_life.give.dialog"},
+         }
+      },
+      end_life_end = {
+         text = {
+            {"progress.end_life.end"},
+         },
+         on_finish = function(t)
+            World.add_deferred_event(20, t.speaker.index)
+         end
+      },
+      end_life_leave = {
+         text = {
+            {"progress.end_life.leave"},
+         }
+      },
+
+      very_late = {
+         text = {
+            {"progress.very_late"},
+         }
+      },
+      late = {
+         text = {
+            {"progress.late", args = function() return {Chara.player().basename} end},
+         }
+      },
+      mid = {
+         text = {
+            {"progress.mid"},
+         }
+      },
+      early = {
+         text = {
+            {"progress.early"},
+         },
+         choices = {
+            {"__END__", "__MORE__"},
```

Comment:
`__MORE__` should be `__BYE__`.
It is a vanilla bug, but it's good time to fix it.

In dialog `last`, `__MORE__` is used too although the dialog does not continue more, but I think it does not need fixing because it's strange that you say "bye" to her according to the content of the talk in `last`.